benchmarks/perception/a3_stereo_image_proc/include/a3_stereo_image_proc/disparity_output_component.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef ROBOTPERF_DISPARITY_OUTPUT_COMPONENT_HPP_
+#define ROBOTPERF_DISPARITY_OUTPUT_COMPONENT_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+#include <image_transport/image_transport.hpp>
+#include <ament_index_cpp/get_resource.hpp>
+#include <stereo_msgs/msg/disparity_image.hpp>
```

Comment:
Typically, `stdlib` and other system headers come first and project headers are last.  Is this convention standard for this project?